### PR TITLE
Changes in bindings workflow

### PR DESCRIPTION
**Pull request by amulyavarote**
Signed-off-by: Amulya Varote <amulyavarote@microsoft.com>

# Description

_Please explain the changes you've made_

Uncommented the workflow for csharp bindings samples. This change will trigger the workflow for csharp bindings examples. 

This PR also includes hello-kubernetes test failures fix.

## Issue reference

We strive to have all PRs being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #738

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] The quickstart code compiles correctly
* [ ] You've tested new builds of the quickstart if you changed quickstart code
* [ ] You've updated the quickstart's README if necessary
* [ ] If you have changed the steps for a quickstart be sure that you have updated the automated validation accordingly. All of our quickstarts have annotations that allow them to be executed automatically as code. For more information see [mechanical-markdown](https://github.com/dapr/mechanical-markdown#readme). For user guide with examples see [Examples](https://github.com/dapr/mechanical-markdown/tree/main/examples#mechanical-markdown-by-example).

### CODE REVIEW

File: tutorials/hello-kubernetes/README.md [hunk 1]
```diff
@@ -303,7 +303,7 @@ Get the API call logs of the node app:
 
 <!-- STEP
 expected_stdout_lines:
-  - 'level=info msg="HTTP API Called: POST /v1.0/state/statestore"'
+  - 'level=info msg="HTTP API Called: POST /v1.0/state/statestore'
```
Comment (paulyuk) [inline tutorials/hello-kubernetes/README.md:306]: extra changes made it in.  these ok?

File: tutorials/hello-kubernetes/README.md [hunk 2]
```diff
@@ -326,7 +326,7 @@ Get the API call logs of the Python app:
 
 <!-- STEP
 expected_stdout_lines:
-  - 'level=info msg="HTTP API Called: POST /neworder"'
+  - 'level=info msg="HTTP API Called: POST /neworder'
```
Comment (paulyuk) [inline tutorials/hello-kubernetes/README.md:329]: extra changes made it in.  these ok?

**Review by paulyuk — CHANGES_REQUESTED** [review]

File: tutorials/hello-kubernetes/README.md (hunk 2 above)
Comment (paulyuk) [reply tutorials/hello-kubernetes/README.md:329]: Also could you please add a sentence or two about what this is, for posterity :)

File: tutorials/hello-kubernetes/README.md (hunk 1 above)
Comment (amulyavarote) [reply tutorials/hello-kubernetes/README.md:306]: Yes!

File: tutorials/hello-kubernetes/README.md (hunk 2 above)
Comment (amulyavarote) [reply tutorials/hello-kubernetes/README.md:329]: Yes!

**Review by paulyuk — APPROVED** [review]
LGTM -- needed for test coverage

**Pull request merged**